doc/dev-lxd.md

```diff
@@ -98,6 +100,38 @@ Return value:
 
     blah
 
+#### `/1.0/devices`
```

Comment:
Why can't `user.network-config` be used for this?

Reply:
`user.network-config` is usually used for a full YAML based description of the network configuration expected to happen inside the container which pretty much covers this case and so shouldn't require adding a devices API to devlxd for this.

I'm not against `ipv4.address` and `ipv6.address` on the nic being used for the address since that part is indeed handled by LXD, but the DNS setting definitely does belong in user.network-config and not as an interface attribute.

Reply:
I agree with DNS settings, it should be common for all interfaces and best place for it is indeed in `user.network-config` as optional thing,
Will remove all DNS related setting and report back